### PR TITLE
completed css-flexbox exercise

### PR DESCRIPTION
**Pull request by AnthonyUrbina**
Screen Shot 2022-08-24 at 4 46 25 PM
Screen Shot 2022-08-24 at 4 46 52 PM
Screen Shot 2022-08-24 at 4 47 06 PM
Screen Shot 2022-08-24 at 5 14 38 PM

### CODE REVIEW

File: css-flexbox/pokemon.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+.pokemon-card img {
+  width: 100%;
+  background-color: lightgray;
+}
+
+#card-deck {
+  display: flex;
+  flex-wrap: wrap;
+}
```
Comment (thebearingedge) [inline css-flexbox/pokemon.css:16]: You can remove this.

File: css-flexbox/reset.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+#card-deck {
+  display: flex;
+  flex-wrap: wrap;
+}
```
Comment (thebearingedge) [inline css-flexbox/reset.css:13]: And this.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great, just a tiny bit of cleanup!

File: css-flexbox/reset.css (hunk 2 above)
Comment (AnthonyUrbina) [reply css-flexbox/reset.css:13]: hmm. i just went and tried to delete the 2 but they were already removed

Comment (AnthonyUrbina) [reply css-flexbox/reset.css]: alright 
both fixed

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**